src/releasers/python.ts

```diff
@@ -48,6 +48,8 @@ const CHANGELOG_SECTIONS = [
 ];
 
 export class Python extends ReleasePR {
+  preRelease = true;
```

Comment:
It's still unclear to me what this member variable is meant to convey.

Reply:
what if we go with the name `enableSimplePrereleaseParsing`, when we undertake your refactor that divides version bumping logic out into its own methods, we can make this one of the strategies.